crypto/store/store_lib.c

```diff
@@ -513,7 +513,14 @@ static int ossl_store_close_it(OSSL_STORE_CTX *ctx)
         ret = ctx->loader->p_close(ctx->loader_ctx);
 #ifndef OPENSSL_NO_DEPRECATED_3_0
     if (ctx->fetched_loader == NULL)
+# if defined (OPENSSL_TANDEM_FLOSS) && defined (_SPT_MODEL_)
```

Comment:
This would be better sited up the top of the file.  There are no calls to `close(2)` in this file.

Reply:
Uhmm @paulidale, a couple of lines down, you have this:

``` C
         ret = ctx->loader->close(ctx->loader_ctx);
```

Reply:
I know, it's just that this kind of rewriting is out of place here.  Also so the fix up afterwards.

Since there is no other instance of the word _close_ in this file, it doesn't hurt to fix this up the top with an explanatory comment.  That where this kind of kludge is oft put.

Reply:
Yes, I agree that it's a bit misplaced, and also that the re-definition after isn't necessary

Reply:
... and now that I've re-read your first review comment...  I read it differently half an hour ago, so never mind my blabbering.

Reply:
(I may need more coffee... yeah, that's probably it)

Reply:
or more sleep :wink:

Reply:
also why not just `# if defined(close)` ? (but keep the comment of course)

Reply:
I have used the suggestion @t8m made, being the simplest to do and understand.

Reply:
Unfortunately this breaks the Windows build. A proper thing would be to rename the structure member as we have this redefinition on Windows as well but it apparently applies in the headers declaring the struct as well which for some reason does not on NonStop.